internal/pipe/changelog/changelog_test.go

```diff
@@ -814,12 +840,23 @@ func TestGroup(t *testing.T) {
 		},
 	}, testctx.WithCurrentTag("v0.0.2"), withFirstCommit(t))
 	require.NoError(t, Pipe{}.Run(ctx))
-	require.Contains(t, ctx.ReleaseNotes, "## Changelog")
-	require.Contains(t, ctx.ReleaseNotes, "### Bots")
-	require.Contains(t, ctx.ReleaseNotes, "### Features")
-	require.Contains(t, ctx.ReleaseNotes, "### Bug Fixes")
-	require.NotContains(t, ctx.ReleaseNotes, "### Catch nothing")
-	require.Contains(t, ctx.ReleaseNotes, "### Others")
+	require.Regexp(t, `## Changelog
+### Features
+\* \w+ feat: added that thing
+### Bug Fixes
+\* \w+ bug: Merge pull request #999 from goreleaser\/some-branch
+### Bots
+\* \w+ feat\(deps\): update foobar \[bot\]
+### Others
+\* \w+ first
+\* \w+ this is not a Merge pull request
+\* \w+ chore: something about cArs we dont need
+\* \w+ docs: whatever
+\* \w+ fix: whatever
+\* \w+ ignored: whatever
+\* \w+ fixed bug 2
+\* \w+ added feature 1
+`, ctx.ReleaseNotes)
```

Comment:
I updated this section to make sure the sorting of changelog groups is tested. `require.Contains` does not ensure the order of the changelog groups.

https://github.com/goreleaser/goreleaser/blob/d2469666b8fe4c14bbce4c7a58f557350fd19d64/internal/pipe/changelog/changelog.go#L214-L221